### ISSUE TITLE
On openSUSE, installing openh264 extension fails

### ISSUE THREAD

**Issue by slaffcheff** [flatpak/flatpak#3155]
## Linux distribution and version
openSUSE Leap 15.0
## Flatpak version
```
Information for package flatpak:
--------------------------------
Repository     : openSUSE-Leap-15.0-Update                  
Name           : flatpak                                    
Version        : 0.10.4-lp150.8.1                           
Arch           : x86_64                                     
Vendor         : openSUSE                                   
Installed Size : 2.4 MiB                                    
Installed      : Yes                                        
Status         : up-to-date                                 
Source package : flatpak-0.10.4-lp150.8.1.src               
Summary        : OSTree based application bundles management
Description    :                                            
    flatpak is a system for building, distributing and running sandboxed desktop
    applications on Linux. See https://wiki.gnome.org/Projects/SandboxedApps for
    more information.

```

## Description of the problem
The "latest" openSUSE version is actually really outdated version. When I try to update flatpak apps I get the following error:
```
Looking for updates...
error: runtime/org.freedesktop.Platform.openh264/x86_64/19.08 needs a later flatpak version (1.4.2;1.2.5;1.0.9;)
```

## Steps to reproduce
Just install flatpak on openSUSE, put couple of apps and try to update it after that. I removed it (with dependencies) reinstalled it and it's exactly the same. Flatpak in unusable for me right now. THe apps are working but cannot update them.

**Comment by matthiasclasen** [comment]
Flatpak 0.10 is very old. As the error message says, we support stable branches back to 1.0.x

**Comment by slaffcheff** [comment]
> Flatpak 0.10 is very old. As the error message says, we support stable branches back to 1.0.x

Cool, but as you can see it says "UP TO DATE" in package info. It's not an issue from my end.

**Comment by refi64** [comment]
LEAP 15.0 is going to EOL in under a month, and 15.1 has a much newer
Flatpak version (1.2.3 w/ the CVE fix from 1.2.4 backported). However, it's
still too old to use the extra-data support in extensions that the runtime
19.08 requires. Please contact your distro for this.

**Comment by mwleeds** [comment]
Closing this since it's openSUSE's decision which flatpak version to package.
